com.github.Alcaro.Flips.json

```diff
@@ -0,0 +1,34 @@
+{
+    "app-id" : "com.github.Alcaro.Flips",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flips",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* OpenGL + DRI access */
+        "--device=dri",
+        /* Needed to find the ROMs and patches */
+        "--filesystem=host:ro"
+    ],
+    "modules" : [
+        {
+            "name" : "flips",
+            "buildsystem" : "simple",
+            "build-commands": [
+                "sh make.sh --cflags=-DFLATPAK",
+                "make install PREFIX=/app"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/Alcaro/Flips.git"
+                }
```

Comment:
cool, so all we're waiting for here is a release/tag/blessed hash from upstream that includes the changes.

Reply:
Alcaro doesn't want to release this version before years and finds tagging a random commit a bit silly, he is fine with releasing it with us using the commit that is currently on master though, so I pinned the repo to that specific commit. Hoping it's good. :)

Reply:
if upstream considers that 'blessed' then sure.